Create fireside chat chapter (#2323)
alan-turing-institute/the-turing-way: book/website/community-handbook/fireside-chat.md

(ch-fireside-chat)=
# Fireside Chat Series

*The Turing Way* Fireside Chat Series features public events with experts, champions and their projects in reproducibility, open research, ethics, collaboration and other opportunities in data science and research.

*The Turing Way* team collaborates with different international communities to co-design and co-host Fireside Chat events that are online hosted informal discussions on topics of shared interests across research communities. 
Speakers are invited from diverse backgrounds and perspectives to present their views, contextualise the topics in their areas of work and catalyse cross-community collaborations through knowledge sharing.

```{figure} ../figures/fireside-chat.jpg
---
name: fireside-chat
alt: People sitting around fire and talking to each other in an informal way.
---
Fireside Chat in The Turing Way.
_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
```

## Background

> ```{admonition} Definition and history

Author: ```suggestion
```{admonition} Definition and History
```

*The Turing way* Fireside Chat series will facilitate discussions on topics of broader interest, create shared spaces for cross-community collaborations and build opportunities for exchanging common experiences and resources. 

## Scope

Fireside Chat series is hosted online on topics of interest within *The Turing Way* community as well as members across other mission-aligned projects such as Open Life Science, The Carpentries, Open Post Academics, Open Science Community in Saudi Arabia, Wikimedia, Frictionless Data, Talarify in South Africa and MetaDocencia (some of our collaborators so far).
We will continue to collaborate with other communities working in participatory, open research and citizen space to organise future events on topics that they are working on.

Although, *The Turing Way* hosts these calls in English, we encourage our collaborative projects to consider hosting a future event in their time zones, in their primary languages and on topics relevant to their local contexts.

### Themes

> Titles of past events include "Project, Community, Reproducible Research: What Exactly is The Turing Way?", "Shared Concerns in Open Research Communities", "Gaps and opportunities for multilingual data science" and "Emergent Roles in Research Infrastructure & Technology".

Author: ```suggestion
Titles of past events include "Project, Community, Reproducible Research: What Exactly is The Turing Way?", "Shared Concerns in Open Research Communities", "Gaps and opportunities for multilingual data science", "Emergent Roles in Research Infrastructure & Technology" and "Enabling inclusive research conferencing in the 'post-pandemic era'".
```

Suggestions of topics for the future Fireside chat include:
1) Hidden labour in open source: How are we changing volunteer culture?
3) Planning sustainability for our grassroots projects
4) Cross-community resource exchange: learning from citizen spaces

> 

Author: ```suggestion
5) Harnessing potential for research communication in 21st century
6) Challenges for creating open (research) infrastructures
```

- **Shared Documents**: We currently use Etherpad by Software Freedom Conservancy to enable shared note-taking during the event. See this for example - https://pad.sfconservancy.org/p/ttw-fireside-chat-mar2022. 
    - **Template** is provided as a subchapter: {ref}`ch-template-fireside-chat`.
    - **Archived notes** are currently shared via HackMD: https://hackmd.io/@turingway/fireside-chats.
- **Video Recording**: Video recordings with closed captioning are shared on *The Turing Way* YouTube channel in the [Fireside Chat video playlist](https://www.youtube.com/watch?v=nuNA3Qa8A-k&list=PLBxcQEfGu3DmBcSmt9GVfo4wN1KP1y2yl).

### Participants

- **Hosts**: Two co-hosts, one from The Turing Way team and one from the collaborating project drive the planning and designing of the event, as well as co-chairing the event. 
- **Speakers**: 3-4 speakers are invited from multiple communities to allow multiple perspectives to be shared. Speakers are invited several weeks before the event to contribute to the event at the planning phase by sharing questions they hope to discuss during the event.

> - **Open participation**: To enable open collaboration during the event, we facilitate open collaboration via Zoom chat and shared note-taking. Although the event is an informal discussion, we record the session upon speakers' permission. We do not run interactive sessions to avoid recording anyone who may not have consented to be recorded. Call hosts can plan to allocate part of the event for open discussions that should not be recorded.

Reviewer: would suggest "via Zoom chat and shared note-taking on Etherpad".

Author: ```suggestion
- **Open participation**: To enable open collaboration during the event, we facilitate open collaboration via Zoom chat and shared note-taking in Etherpad. Although the event is an informal discussion, we record the session upon speakers' permission. We do not run interactive sessions to avoid recording anyone who may not have consented to be recorded. Call hosts can plan to allocate part of the event for open discussions that should not be recorded.
```

> *The Turing Way* Code of Conduct applies to the Fireside Chat participants and all interactions taking place as a part of the event.

Author: ```suggestion
*The Turing Way* {ref}`Code of Conduct<ch-coc>` applies to the Fireside Chat participants and all interactions taking place as a part of the event.
```

```

### Fireside Chats in _The Turing Way_

Through the Fireside Chat series in *The Turing Way*, we want to build an intentional space to gather, facilitate important conversations and establish cross-community collaborations.

Different stakeholders in open research and grassroots communities have a shared mission of making scientific knowledge freely available for public access and for the benefit of society. 
However, most of these communities operate independently of other initiatives, either lacking the capacity to build meaningful collaboration or competing for limited resources. 
This often results in scientific outputs that most users can find, but not access, or build upon in their local contexts.
Open discussions and spaces for collaborations can contribute to individual benefit, team efforts as well as the overall sustainability of the open science ecosystem {cite}`Sharan2021reimagine`.

> Through collaboratively organised Fireside Chat discussions, we create dedicated opportunities crucial for exchanging experiences and resources. 
> With the Fireside Chat series, we aim to enable discussions that are of broader interest and can enable cross-community interactions and exchanges.

Author: ```suggestion
*The Turing way* Fireside Chat series will facilitate discussions on topics of broader interest, create shared spaces for cross-community collaborations and build opportunities for exchanging common experiences and resources. 
```

## Scope

> Fireside Chat series is hosted online on topics of interest within *The Turing Way* community as well as members across other mission-aligned projects such as Open Life Science, The Carpentries, Open Post Academics, Open Science Community in Saudi Arabia, Wikimedia and Talarify in South Africa and MetaDocencia.

Author: ```suggestion
Fireside Chat series is hosted online on topics of interest within *The Turing Way* community as well as members across other mission-aligned projects such as Open Life Science, The Carpentries, Open Post Academics, Open Science Community in Saudi Arabia, Wikimedia, Frictionless Data, Talarify in South Africa and MetaDocencia (some of our collaborators so far).
```
